Response to reviewers (#38)
11michalis11/ED-EMS-game-paper: reviews/EJOR/01/main.tex

\documentclass{article}

\title{Response to reviewers}
\author{}

\begin{document}
    \maketitle

    I would like to thank all the reviewers for their help and comments and for

>     the time they took to overhaul the manuscript.

Reviewer: ```suggestion
    the time they took to review the manuscript.
```

>     Their suggestions were extremely helpful and have been considered to improve

Reviewer: ```suggestion
    Their suggestions were extremely helpful and we believe this has resulted in an improved
```

>     the manuscript.

Reviewer: ```suggestion
    manuscript.
```

(Doing this on my tablet and the GitHub app doesn't allow for multi line suggestions I think)

>     The following changes were made:

Reviewer: ```suggestion
    Below is a point by point response and summary of changes made.
```

        motivation in Section 1, and to put the literature review in a
        separate Section.}
    \end{itemize}
    The paper has been completely restructured.
    Sections 2 and 5 have been combined and the game theoretic model is now
    presented along with the results in one section.
    In addition, section 1 contains only the introduction while the
    literature review has moved to section 2.
    Finally, the queueing model section now comes before the game theoretic
    section.
    The current structure is: Introduction; Literature Review; A queueing
    model for the ED-EMS interface; Strategic manipulation of the ED-EMS
    interface; Results.

    \line(1, 0){0.92\textwidth}
    \begin{itemize}
        \item Reviewer 3 - Comment 2: \textit{Since the application in mind
        is the EMS-ED interface, it would be better to write the paper
        immediately from this perspective, instead of first abstracting it
        and then applying it to that problem.}
        \item Reviewer 3 - Comment 8: \textit{Figs 7-11: Row player and
        Column Player are unclear: hospital 1 and hospital 2 would be a
        better choice I presume.}
    \end{itemize}

>     The paper is now written more application focused.

Reviewer: ```suggestion
    The paper is presented with the application of the model as the main focus.
```

    All sections are now presented from the perspective of the EMS-ED
    interface.
    In the game theoretic section instead of row/column player we now use
    hospital A and hospital B.

    \line(1, 0){0.92\textwidth}
    \begin{itemize}
        \item Reviewer 1 - Comment 4: \textit{To highlight the contributions
        of this work, it is necessary to compare the novelty of this work
        with previous results in the literature.}
    \end{itemize}

>     In the new literature review section we now compare the results from

Reviewer: ```suggestion
    In the new literature review section we now directly compare the results from
```

    previous work with the results from this work to highlight the
    contributions of this work.

    \line(1, 0){0.92\textwidth}
    \begin{itemize}
        \item Reviewer 3 - Comment 6: \textit{Figure 3: about solving the
        Markov chain: the Markov chain is truncated to solve it. Therefore,
        probably N and M are not too small. What about the numerical
        complexity? Because if you just solve it, you have to invert the
        matrix? Is there some structure in the Markov chain that can be
        exploited if N and M would be large?}
        \item Reviewer 3 - Comment 7: \textit{Figures 4-6: why do you
        compare your calculated values with simulation?
        Is it to evaluate the truncations?
        Or are you not sure about the formulas?}
    \end{itemize}

>     This is a good observation.

Reviewer: ```suggestion
    This observation was really helpful and lead to us conducting more numerical experiments to clarify the point of truncating the state space.
```

    As the values of \(N\) and \(M\) increase, the truncated Markov chain
    becomes more complex to solve.

>     We have touched upon building a closed-form formula for the steady state
>     probabilities but it's still work in progress.

Reviewer: ```suggestion
    We have touched upon building a closed-form formula for the steady state
    probabilities but that is still a work in progress and not something we are able
    to include in this manuscript.
```

>     Figures 4-6 are there to demonstrate the accuracy of our constructed
>     model against its discrete event simulation equivalent.

Reviewer: ```suggestion
    Figures 4-6 are there to demonstrate the accuracy of our constructed
    model against its discrete event simulation equivalent and these have been expanded on.
```

Is that right?

    An additional paragraph is added that talks about truncating the model
    and the old figures have been replaced with ones that compare the
    results from the truncated Markov model with the results from the

>     untruncated simulation model.

Reviewer: ```suggestion
    untruncated simulation model. We feel that these extra numeric results 
    do clarify the paper and are thankful for the reviewer comments.
```
